### PR TITLE
[CCXDEV-7074] Sync openapi.json for v2 with iqe-ccx-plugin

### PR DESCRIPTION
**Pull request by matysek**
# Description

Sync openapi.json for v2 with iqe-ccx-plugin that is used for api/v2 fuzz tests.

### CODE REVIEW

**Review by Bee-lee — APPROVED** [review]
I trust you with this one :D

File: server/api/v2/openapi.json [hunk 1]
```diff
@@ -36,87 +40,68 @@
...
           }
-        },
-        "operationId": "getMetrics",
-        "summary": "Read all metrics exposed by this service",
-        "description": "Set of metrics provided by insights-operator-utils library providing total number of requests counter, API endpoints response times, and counter of HTTP status code responses. Additionally it is possible to consume all metrics provided by Go runtime. These metrics start with go_ and process_ prefixes."
+        }
       }
     },
-    "/cluster/{clusterId}/reports": {
+    "/clusters/{clusterList}/reports": {
```
[the remark this answers is not in the document]
Comment (matysek) [reply server/api/v2/openapi.json:70]: @joselsegura checking.

[the remark this answers is not in the document]
Comment (matysek) [reply server/api/v2/openapi.json]: @joselsegura fixed to /cluster/.

**Review by tisnik — APPROVED** [review]
ok

**Pull request merged**